docs/user-guide/horizontal-pod-autoscaling/index.md

```diff
@@ -10,20 +10,21 @@ This document describes the current state of Horizontal Pod Autoscaling in Kuber
 ## What is Horizontal Pod Autoscaling?
 
 With Horizontal Pod Autoscaling, Kubernetes automatically scales the number of pods
-in a replication controller, deployment or replica set based on observed CPU utilization
-(or, with alpha support, on some other, application-provided metrics).
+in a replication controller, deployment or replica set based on observed CPU utilization.
+Alternatively, Kubernetes 1.2 added  alpha support for scaling based on application-provided metrics 
+via cAdvisor.
```

Comment:
According to the new architecture we've been discussing it will not have to be just cAdvisor, but any monitoring pipeline. See https://github.com/kubernetes/kubernetes/pull/34758. @DirectXMan12 is now working on exposing custom metrics to HPA and adding support in the API itself.